samcli/commands/_utils/custom_options/replace_help_option.py

```diff
@@ -0,0 +1,12 @@
+import click
+
+
+class ReplaceHelpSummaryOption(click.Option):
+    def __init__(self, *args, **kwargs):
+        self.replace_help_option = kwargs.get("replace_help_option", "")
+        kwargs.pop("replace_help_option")
+        super(ReplaceHelpSummaryOption, self).__init__(*args, **kwargs)
+
+    def get_help_record(self, ctx):
+        option, help_text = super(ReplaceHelpSummaryOption, self).get_help_record(ctx=ctx)
```

Comment:
I'm not fully understanding the reason for overriding the help summary option and this method. Likely just missing something.

Reply:
Otherwise it looks like below:

![Screen Shot 2023-03-13 at 2 59 33 PM](https://user-images.githubusercontent.com/3770774/224841868-130d6b8d-bcd8-4643-8753-ad7e1441f3dd.png)